Mathmagician.Tests/Mathmagician.Tests.csproj

```diff
@@ -57,7 +57,11 @@
     <Compile Include="UnitTest1.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="IntegerTests.cs" />
+<<<<<<< HEAD
```

Comment:
Is this from a merge conflict? If so, does this build with the <<<<<<HEAD in it?